go-controller/pkg/node/gateway_udn.go

```diff
@@ -43,6 +43,12 @@ type UserDefinedNetworkGateway struct {
 	// masqCTMark holds the mark value for this network
 	// which is used for egress traffic in shared gateway mode
 	masqCTMark uint
+	// stores a copy of default network's gateway so that
```

Comment:
I dont think this is a copy. It looks like the pointer is passed all the way to this struct.

Reply:
true, wrong wordings.. changing this.